app/javascript/src/Navigation/components/header/DocumentationMenuItem.jsx

```diff
@@ -1,6 +1,15 @@
-import React from 'react'
+// @flow
 
-const DocumentationMenuItem = ({docsLinksClass, item}) => (
+import * as React from 'react'
+
+type ItemType = {
```

Comment:
I prefer this way over importing types from another file when it's just used in one component. That way you can see at glance what is going on.